tests/zeroconfig/windows/testdata/resource_traces/aspnetcore.yaml

```
@@ -21,3 +21,4 @@ resource_spans:
         net.peer.ip: <ANY>
         signalfx.tracing.library: dotnet-tracing
         signalfx.tracing.version: <ANY>
+        splunk.zc.method: <ANY>
```

fwiw this should support something like

```suggestion
        splunk.zc.method: <RE2(signalfx-dotnet-tracing-.*)>
```

---

Thanks! Was wondering if it supported that.

---

Sorry, I didn't add the default value loader to the ResourceTraces so this is untrue atm.